manager/integration/deploy/test.yaml

```diff
@@ -42,7 +42,7 @@ spec:
     - name: LONGHORN_JUNIT_REPORT_PATH
       value: /tmp/test-report/longhorn-test-junit-report.xml
     - name: LONGHORN_BACKUPSTORES
-      value: "s3://backupbucket@us-east-1/backupstore$minio-secret, nfs://longhorn-test-nfs-svc.default:/opt/backupstore"
+      value: "s3://backupbucket@us-east-1/backupstore$minio-secret, nfs://longhorn-test-nfs-svc.default:/opt/backupstore, cifs://longhorn-test-cifs-svc.default/backupstore$cifs-secret, azblob://longhorn-test-azurite@core.windows.net/$azblob-secret"
```

Comment:
Are we supporting selecting any of the backupstore in Jenkins and test will run only with that backupstore?

Reply:
Yes we can set  `nfs`, `s3`(current design) `cifs`  `azurite` (in this PR) for single backupstore for test, all other strings will run all backupstores.